src/styles/bramble_overrides.css

```diff
@@ -86,13 +111,16 @@
     background: transparent !important;
 }
 
-
 /* Makes the resize handles wider, and visile on hover & drag */
 
 #sidebar {
   position: relative;
 }
 
+.CodeMirror-foldgutter:after {
```

Comment:
During a merge conflict I just appended it under my stuff. I wasn't sure what its used for. So I kept it there.